src/main/java/org/mockito/ArgumentMatcher.java

```diff
@@ -89,17 +89,24 @@
  *
  * All existing custom implementations of <code>ArgumentMatcher</code> will no longer compile.
  * All locations where hamcrest matchers are passed to <code>argThat()</code> will no longer compile.
- * There are 2 approaches to fix the problems:
+ * There are 3 approaches to fix the problems:
  * <ul>
- * <li>a) Refactor the hamcrest matcher to Mockito matcher:
+ * <li>Refactor the hamcrest matcher to Mockito matcher:
  * Use "implements ArgumentMatcher" instead of "extends ArgumentMatcher".
  * Then refactor <code>describeTo()</code> method into <code>toString()</code> method.
  * </li>
+ * <li>If you need to separate the changes to the code from the switching of the Mockito versions,
```

Comment:
Given that the code changed and now we have a separate 'migration' module, can you update this part of the javadoc. I really like that you added option no. 3 to the list. Perhaps it's enough if the new option is one sentence / paragraph and a link to the compatibility matcher.